SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift

```diff
@@ -8,15 +8,43 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
     static let valueCellId = "valueCellId"
   }
 
+    enum Priority: String {
```

Comment:
This block isn't formatted properly. Please run `clang-format-changed-files.sh` script in the root folder of the repo.

Reply:
Sorry for confusion. The script only works for Objective C files.

Reply:
I fixed indentations for that, please check.